Sources/Rendering/Core/VolumeMapper/index.d.ts

```diff
@@ -94,6 +94,24 @@ export interface vtkVolumeMapper extends vtkAbstractMapper {
 	 */
 	getAnisotropy(): number;
 
+	/**
+	 * Get local ambient occlusion flag
+	 * @default false
+	 */
+	getLocalAmbientOcclusion(): boolean;
+
+	/**
+	 * Get kernel size for local ambient occlusion
+	 * @default 15
+	 */
+	getKernelSize(): number;
+
+	/**
+	 * Get kernel radius for local ambient occlusion
+	 * @default 7
+	 */
+	getKernelRadius(): number;
+
```

Comment:
How about `getLAOKernelSize` and `getLAOKernelRadius` for clarity? (or `getLaoKernelSize` if you want the model property name to be `laoKernelSize`).

Reply:
Sounds good. Just changed the names.